docs-2.0/nebula-console.md

```diff
@@ -1,14 +1,55 @@
 # Nebula Console
 
-Nebula Console is a native CLI client for Nebula Graph. It can be used to connect a Nebula Graph cluster and execute queries. It can also support special commands to manage parameters, export query results, import test datasets, etc. This topic describes these special commands.
+Nebula Console is a native CLI client for Nebula Graph. It can be used to connect a Nebula Graph cluster and execute queries. It also supports special commands to manage parameters, export query results, import test datasets, etc.
 
-To connect Nebula Graph using Nebula Console, see [Step 3: Connect to Nebula Graph](2.quick-start/3.connect-to-nebula-graph.md).
+## Obtain Nebula Console
 
-!!! note
+You can obtain Nebula Console in the following ways:
+
+- Obtain the binary file of Nebula Console from the `bin` directory in the Nebula Graph installation path.
+
+- Download the binary file from the [GitHub releases page](https://github.com/vesoft-inc/nebula-console/releases "the nebula-console Releases page").
+
+- Compile the source code to obtain the binary file. For more information, see [Install from source code](https://github.com/vesoft-inc/nebula-console#from-source-code).
+
+## Nebula Console functions
+
+### Connect to Nebula Graph
+
+To connect to Nebula Graph with the `nebula-console` file, use the following syntax:
+
+```bash
+<path_of_console> -addr <ip> -port <port> -u <username> -p <password>
+```
 
-    The commands are case insensitive.
+`path_of_console` indicates the storage path of the Nebula Console binary file.
 
-## Manage parameters
+Parameter descriptions are as follows:
+
+| Parameter | Description |
+| - | - |
+| `-h/-help` | Shows the help menu. |
+| `-addr/-address` | Sets the IP address of the Graph service. The default address is 127.0.0.1. If Nebula Graph is deployed on [Nebula Cloud](https://docs.nebula-graph.io/2.6.2/nebula-cloud/1.what-is-cloud/), you need to create a Private Link and set the IP address of the Private Endpoint as the parameter value. |
+| `-P/-port` | Sets the port number of the graphd service. The default port number is 9669. |
+| `-u/-user` | Sets the username of your Nebula Graph account. Before enabling authentication, you can use any existing username. The default username is `root`. |
+| `-p/-password` | Sets the password of your Nebula Graph account. Before enabling authentication, you can use any characters as the password. |
+| `-t/-timeout`  | Sets an integer-type timeout threshold of the connection. The unit is second. The default value is 120. |
+| `-e/-eval` | Sets a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned. |
+| `-f/-file` | Sets the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. The result will be returned and the connection stops then. |
+| `-enable_ssl` | Enable SSL encryption when connecting to Nebula Graph. |
```

Comment:
is it better to unify the singular or plural forms of the first word? Please also check the following sentences.

Reply:
Good catch!